client/Assets/Scripts/Game/MatchmakingBehaviour.cs

```diff
@@ -42,43 +42,43 @@ public async void FindGame()
 			await _beamContext.OnReady;
 
 			Debug.Log($"Starting matchmaking with game_type={GameTypeRef.Id} override timeout={_configuration.OverrideMaxMatchmakingTimeout} ...");
-			var handle = await _beamContext.Api.Experimental.MatchmakingService.StartMatchmaking(GameTypeRef.Id);
+			// var handle = await _beamContext.Api.Experimental.MatchmakingService.StartMatchmaking(GameTypeRef.Id);
```

Comment:
Let's avoid leaving commented-out lines in the code unless they are needed for illustrative purposes (and have lots of comments explaining them 😺). In this case, it sounds like the double call to StartMatchmaking was a leftover mistake.

I'd say open another small PR to delete this commented-out line.